docs/lib/predictions/fragments/android/identify-celebrity.md

```diff
@@ -0,0 +1,50 @@
+## Set up your backend
+
+If you haven't already done so, run `amplify init` inside your project and then `amplify add auth` (we recommend selecting the *default configuration*).
+
+Run `amplify add predictions`, then use the following answers:
+
+```console
+? Please select from one of the categories below (Use arrow keys)
+❯ Identify
+  Convert
+  Interpret
+  Infer
+  Learn More
+
+? What would you like to identify? (Use arrow keys)
+  Identify Text
+❯ Identify Entities
+  Identify Labels
+
+? Provide a friendly name for your resource
+  <Enter a friendly name here>
+
+? Would you like use the default configuration? (Use arrow keys)
+❯ Default Configuration
+  Advanced Configuration
+
+? Who should have access? (Use arrow keys)
+  Auth users only
+❯ Auth and Guest users
+
+```
+Run `amplify push` to create the resources in the cloud
+
+## Working with the API
+
+```java
+public void detectCelebs(Bitmap image) {
+    Amplify.Predictions.identify(
+            IdentifyActionType.DETECT_CELEBRITIES,
+            image,
+            result -> {
+                IdentifyCelebritiesResult identifyResult = (IdentifyCelebritiesResult) result;
+                CelebrityDetails metadata = identifyResult.getCelebrities().get(0);
+                Log.i("PredictionsQuickstart", metadata.getCelebrity().getName());
+            },
+            error -> Log.e("PredictionsQuickstart", error.toString(), error)
+    );
+}
+```
+As a result of passing in an image of a celebrity, you should see the corresponding celebrity name printed to the console.
```

Comment:
Should or will?

Reply:
I think should works nicely here, mostly because if they pass in a celebrity we are unaware of then they might not see a celebrity name.

Reply:
That can be phrased much more precisely:

> As a result of passing in an image, `identify` will return the name of a celebrity if detected.